src/components/MAPI/apps/ClusterDetailApps.tsx

```diff
@@ -142,24 +179,45 @@ const ClusterDetailApps: React.FC<
             </Box>
           </ClusterDetailAppList>
 
-          <h3>Preinstalled Apps</h3>
-          <Disclaimer margin={{ bottom: 'medium' }} fill={true}>
-            These apps are preinstalled on your cluster and managed by Giant
-            Swarm.
-          </Disclaimer>
-
-          <ClusterDetailReleaseApps
-            appList={appList}
-            isLoading={appListIsLoading}
-            isClusterCreating={isClusterCreating}
-            errorMessage={extractErrorMessage(appListError)}
-            appsPermissions={appsPermissions}
-            releaseVersion={releaseVersion}
-          />
-
-          <Box margin={{ top: 'medium' }} direction='column' gap='small'>
-            <ListAppsGuide namespace={clusterId} />
-          </Box>
+          {typeof isClusterApp !== 'undefined' &&
+            typeof clusterVersion !== 'undefined' && (
```

Comment:
Is it possible to use `isClusterApp && clusterVersion` here, or would it affect the UI loading behaviour?

Reply:
I can change it to
```
typeof isClusterApp !== 'undefined' && clusterVersion
```

`isClusterApp` can be `true | false | undefined`, in this case only `undefined` is not allowed.

Reply:
Thanks for explaining! It's not a blocker from me, just wondered if we could simplify it a little bit